.github/ISSUE_TEMPLATE.md

```diff
@@ -1,22 +1,24 @@
-**Please DO NOT report security vulnerabilities here. Send them to security@mautic.com instead.**
+[//]: # (READ THIS BEFORE PUBLISHING YOUR ISSUE)
 
-What type of report is this:
+[//]: # (Please post questions, support requests and feature requests/ideas at https://forum.mautic.org.)
+[//]: # (Strictly follow the instruction hereunder, otherwise your issue will be close for lack of information.)
```

Comment:
```suggestion
[//]: # (Strictly follow the instructions hereunder, otherwise your issue will be closed for lack of information.)
```
I'm not sure as I'm not an English man but this seem better. @RCheesley ?

Reply:
Hah, yes good catch, indeed that's much better!